src/components/VotesCounter.jsx

```diff
@@ -0,0 +1,45 @@
+import { useState, useEffect } from 'react';
+import { getArticleVotesCount, updateArticleVotes } from '../api';
+
+const VotesCounter = ({article_id, votes}) => {
+  const [votesCount, setVotesCount] = useState(votes)
```

Comment:
It is a react anti-pattern to initialise state with a prop and could cause some unexpected bugs.
It would be more appropriate to initialise this state as 0 and then render votesCount + votes